Framework/PythonInterface/mantid/api/src/Exports/ITableWorkspace.cpp

```diff
@@ -26,9 +26,16 @@
 #include <boost/python/dict.hpp>
 #include <boost/python/list.hpp>
 #include <boost/python/make_constructor.hpp>
+
 #include <cstring>
 #include <vector>
 
+#if PY_MAJOR_VERSION >= 3
```

Comment:
Can use the already defined stuff from [`VersionCompat`](https://github.com/mantidproject/mantid/blob/master/Framework/PythonInterface/core/inc/MantidPythonInterface/core/VersionCompat.h) here?